### PR TITLE
[nginx] Fix startup crashes

### PR DESCRIPTION
**Pull request by yooyoo9**

### CODE REVIEW

File: projects/nginx/fuzz/http_request_fuzzer.cc [hunk 1]
```diff
@@ -307,15 +307,4 @@ DEFINE_PROTO_FUZZER(const HttpProto &input) {
 
   // Will redirect to http parser
   ngx_http_init_connection(c);
-
-  // Clean-up in case of error
```
Comment (inferno-chromium) [inline projects/nginx/fuzz/http_request_fuzzer.cc:311]: Is this not needed anymore ?

Comment (yooyoo9) [reply projects/nginx/fuzz/http_request_fuzzer.cc:311]: Yes

Comment (oliverchang) [reply projects/nginx/fuzz/http_request_fuzzer.cc:311]: Can you please explain why this is no longer required?

Comment (yooyoo9) [reply projects/nginx/fuzz/http_request_fuzzer.cc:311]: The fuzzer was originally fuzzing the client side of nginx, now that it also fuzzes the server side, it handles the clean-up properly.

**Comment by yooyoo9** [comment]
@inferno-chromium @oliverchang Anything else to change?

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**